app/controllers/publishers/vacancies/feedbacks_controller.rb

```diff
@@ -1,13 +1,18 @@
 class Publishers::Vacancies::FeedbacksController < Publishers::Vacancies::BaseController
+  def new
+    @vacancy = VacancyPresenter.new(Vacancy.published.find(params[:job_id]))
```

Comment:
Not sure if you need `@vacancy` here. 

The vacancy is available as `vacancy`. 

Take a look at `app/controllers/publishers/vacancies/base_controller.rb`